learning_resources/tasks.py

```diff
@@ -209,7 +209,7 @@ def get_content_tasks(  # noqa: PLR0913
     if learning_resource_ids:
         learning_resources = LearningResource.objects.filter(
             id__in=learning_resource_ids, etl_source=etl_source
-        ).values_list("id", flat=True)
+        ).order_by("-id").values_list("id", flat=True)
```

Comment:
Since `else` block below is ordered by `-id`, I decided to do it here too for consistency.